test/python/circuit/test_controlled_gate.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
Maybe a small note here that these tests are coming from Aqua (and the link to the file from where they are coming)?

Reply:
I changed the tests a bit to work with `_compute_control_matrix` so I added `based on the test of Aqua` with a permalink to the current master version 👍